src/Compilers/Core/CodeAnalysisTest/Resources.resx

```diff
@@ -121,15 +121,15 @@
   </resheader>
   <assembly alias="System.Windows.Forms" name="System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089" />
   <data name="defaultWin32Manifest" type="System.Resources.ResXFileRef, System.Windows.Forms">
-    <value>Resources\default.win32manifest;System.Byte[], mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+    <value>Resources/default.win32manifest;System.Byte[], mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
```

Comment:
Why did this need to change?  Can't quite see how msbuild would have affected this.